packages/components/addon/components/hds/form/radio-card/index.js

```diff
@@ -113,16 +83,16 @@ export default class HdsFormRadioCardIndexComponent extends Component {
     if (this.args.disabled) {
       classes.push('hds-form-radio-card--disabled');
     }
+    if (this.args.maxWidth) {
+      classes.push('hds-form-radio-card--fixed-width');
+    }
```

Comment:
Not strongly opinionated. If we do provide two modifiers, it would probably be best to shift to the `--attribute-value` pattern, and I'm not sure how to call this attribute – `--width-fix`, `--width-fuid` – does it read ok?

Reply:
Potentially could be `--has-fixed-width` and `--has-fluid-width` (but would be a bit cheating, I know 😀)

Reply:
works for me